packages/bootstrap-4/src/TextWidget/TextWidget.tsx

```diff
@@ -1,12 +1,10 @@
 import React from "react";
 
-import TextField, {
-  StandardTextFieldProps as TextFieldProps,
-} from "@material-ui/core/TextField";
+import Form from "react-bootstrap/Form";
 
 import { WidgetProps } from "@rjsf/core";
 
-export type TextWidgetProps = WidgetProps & TextFieldProps;
+export type TextWidgetProps = WidgetProps;
```

Comment:
The `textFieldProps` are still being passed and spread onto the Form.Control, so I guess they should also have typings? Not a blocker to move on, but please mark a todo to resolve